src/Commands/config/ConfigCommands.php

```diff
@@ -175,6 +176,85 @@ public function delete($config_name, $options = []) {
   }
 
   /**
+   * Display status of configuration (differences between stored and active configuration).
+   *
+   * @command config-status
+   * @param $label A config directory label (i.e. a key in \$config_directories array in settings.php).
+   * @interact-config-label
+   */
+  public function status($label = NULL) {
```

Comment:
As a follow-up, it would be great to support output formatters in some way for config-status. I think callers will want programmatic access to the $change list.